displayCavity.py

```diff
@@ -1,15 +1,13 @@
-import sys
 from epics import PV
-
-from fault import CSV_FAULTS, Fault, PvInvalid
-
-sys.path.insert(0, '..')
-from constants import STATUS_SUFFIX, SEVERITY_SUFFIX
-from lcls_tools.devices.scLinac import LINACS, Linac, Cavity
 from typing import List
 
+from CavityDisplay import SEVERITY_SUFFIX, STATUS_SUFFIX
```

Comment:
Is this gonna need to be changed to cavityDisplayGUI since we changed the file name?

Reply:
Yup! Just changed it